vercel.json

```diff
@@ -23,7 +23,7 @@
     },
     {
       "source": "/resources/bug-bounty",
-      "destination": "/resources/community/bug-bounty",
```

Comment:
should a redirect be added for this page as well?

Reply:
Added one just in case.